db/fixtures/miq_product_features.yml

```diff
@@ -1932,7 +1932,26 @@
         :feature_type: admin
         :hidden: true
         :identifier: alert_delete
-
+  - :name: Alerts Statuses
```

Comment:
@h-kataria @imtayadeway:
- @jeff-phillips-18 is adding the necessary roles for the screens consuming this API in https://github.com/ManageIQ/manageiq/pull/13357
- should we have duplicate roles for API & UI? only UI? only API?
- current location is wrong: this API isn't under control explorer(fixed in a subsequent [pr](https://github.com/ManageIQ/manageiq/pull/13325/files#diff-bccee92afbf0d3d195b7bcf343b1b6c4) to be in it's own section)